src/components/NewMenuSections/types.ts

```diff
@@ -0,0 +1,21 @@
+import { ResolveOakHrefProps } from "@/common-lib/urls";
+
+type HrefLink = { href: string };
+
+export const linkIsHref = (input: unknown): input is HrefLink => {
+  return typeof (input as HrefLink).href === "string";
+};
+
+export type BetaMenuLink = {
```

Comment:
I wonder if we should change the naming away from beta, as it's not going to exist much longer

Reply:
yeah I agree it could be better named! any suggestions?

Reply:
renamed to `BurgerMenuLink` hopefully that's a bit more descriptive